### PR TITLE
Display draft item in sidebar navigation in prod mode when it is current page

### PR DESCRIPTION
**Pull request by Natalie-Smirnova**
Closes: #1856

### CODE REVIEW

File: pages/11ty/collections.filter.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/** get value from object by path */
+const get = (item, path) => path.split('.').reduce((acc, key) => acc && acc[key], item);
+
+/** filter to include only items that have a property with passed value */
+function filter(collection, prop, value) {
+  const matcher = arguments.length === 2 ?
```
Comment (dshovchko) [inline pages/11ty/collections.filter.js:6]: I'm not sure about the proper understanding of njk, but it seems strange to me.
Do we really have to check the number of arguments passed? And what if there is only one?

Comment (ala-n) [reply pages/11ty/collections.filter.js:6]: That's the way to strictly separate two usecases of the filter:
1) We do not pass the value (third argument in definition, second in filter usage in njk) 
  - `collection | filter('visible')` -  - so we are searching for items with truthy 'visible' field
  - Same with `collection | exclude('visible')` - we are searching for items with falsy 'visible' field
2) We pass the value to the filter:
 - `collection | filter('visible', true)` -  we are searching for items with exact `true` boolean value
 - `collection | exclude('order', 0)` - we are searching for items with order field not equal to 0 (so false, NaN, etc - are acceptable)

**Review by julia-murashko — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

File: pages/11ty/collections.filter.js (hunk 1 above)
Comment (ala-n) [reply pages/11ty/collections.filter.js]: Update the logic to look more in a ES6 style, clarify documentation

**Review by NastaLeo — APPROVED** [review]

**Pull request merged**